src/Microsoft.ML.Data/Transforms/doc.xml

```diff
@@ -13,82 +13,6 @@
         it will keep only the rows that have missing values.
       </remarks>
     </member>
-    <example name="NAFilter">
-      <example>
-        <code language="csharp">
-          pipeline.Add(new MissingValuesRowDropper(&quot;Column1&quot;));
-        </code>
-      </example>
-    </example>
```

Comment:
can you drop starting from TextToKey?
Not what we need this example, I just have PR for missing values and don't want to you struggle with merge.